index.js

```diff
@@ -2,14 +2,14 @@ const { dialog, globalShortcut } = require('electron');
 const getShortcutFromConfig = require('./modules/getShortcutFromConfig');
 
 function registerShortcut (configKey, action) {
-  return (app) => {
+  return (app, ...args) => {
     const { plugins, config } = app;
     function register (accelerator) {
       if (!accelerator) return;
       globalShortcut.unregister(accelerator);
       const registered = globalShortcut.register(
         accelerator,
-        () => action(app)
+        () => action.apply(this, [app, ...args])
```

Comment:
Can this be `() => action(app, ...args)` instead? Using apply implies that the `this` value is important, but I don't think it is here, right?